### PR TITLE
Fix typo in testenv for cross-block local assertion prop

### PR DESCRIPTION
**Pull request by AndyAyersMS**
Also enable for the libraries pgo run.

### CODE REVIEW

**Comment by AndyAyersMS** [comment]
@dotnet/jit-contrib PTAL

File: eng/pipelines/libraries/run-test-job.yml [hunk 1]
```diff
@@ -242,6 +242,7 @@ jobs:
                 - jitosr_stress_random
                 - syntheticpgo
                 - syntheticpgo_blend
+                - jitcrossblocklocalassertionprop
```
Comment (jakobbotsch) [inline eng/pipelines/libraries/run-test-job.yml:245]: Seems like we ought to have a runtime-jit-experimental variant for libraries

**Review by jakobbotsch — APPROVED** [review]

**Comment by AndyAyersMS** [comment]
Going to merge this despite red build analysis, novel failures are in experimental pipeline.

**Pull request merged**